cmd/seednode/main.go

```diff
@@ -147,6 +155,16 @@ func startNode(ctx *cli.Context) error {
 		}
 	}
 
+	restApiPort := ctx.GlobalInt(restApiPortFlag.Name)
+	if restApiPort > 0 {
+		go func() {
```

Comment:
define a function instead of the anonymous one?

Reply:
done. extracted functions